core/arch/arm/arm.mk

```diff
@@ -140,17 +140,18 @@ arm64-platform-cflags-generic := -mstrict-align $(call cc-option,-mno-outline-at
 
 ifeq ($(DEBUG),1)
 # For backwards compatibility
-$(call force,CFG_CC_OPTIMIZE_FOR_SIZE,n)
+$(call force,CFG_CC_OPT_LEVEL,0)
 $(call force,CFG_DEBUG_INFO,y)
 endif
-
-CFG_CC_OPTIMIZE_FOR_SIZE ?= y
-ifeq ($(CFG_CC_OPTIMIZE_FOR_SIZE),y)
-platform-cflags-optimization ?= -Os
-else
-platform-cflags-optimization ?= -O0
+ifeq ($(CFG_CC_OPTIMIZE_FOR_SIZE),n)
```

Comment:
Backward compat with `CFG_CC_OPTIMIZE_FOR_SIZE=n` but not with `CFG_CC_OPTIMIZE_FOR_SIZE=y?`

Reply:
`-Os` is the default value so nothing special is needed I think?

Reply:
Ok, fine as is. Thanks for this.
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`